src/components/nav.js

```diff
@@ -26,9 +31,15 @@ export default function nav() {
         </svg>
       </div>
       <div className="nav-pills">
-        <a href="/auth/login.html" className=" btn-filled">
```

Comment:
use Link tags here as well

Reply:
Link tag needs mandatory `to` attribute, and for this, I didn't need that. so when I was using it without `to`, there were errors.